fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/Calculator.kt

```diff
@@ -146,12 +146,14 @@ class Calculator(activity: Activity,
         val spaceBelow = bitmapHeight - bottom.toInt()
         val params = view.layoutParams as RelativeLayout.LayoutParams
 
+        val viewHeight = if (focusShape == FocusShape.ROUNDED_RECTANGLE) focusHeight else viewRadius
```

Comment:
It should be `focusHeight / 2` 
Because bottom position will be calculated by adding this value to the **center** of the view not the top of the view.

Reply:
done, did not see this mistake...